### ISSUE TITLE
by default, connect straight to localhost:7099

### ISSUE THREAD

**Issue by benswift** [noahingham/extempore-atom#20]
Hey @noiach 

After some [discussion on the mailing list](https://groups.google.com/forum/#!topic/extemporelang/7vaOWmcmvEc) we've changed the default behaviour of the connect command (for Emacs & VSCode). Is there any chance you could update the atom plugin to match this behaviour?

As an aside, if you no longer have the time to maintain this, I'm open to migrating it to the extemporelang org. But the current situation is ok as well if you'd prefer.

**Comment by benswift** [comment]
ping @noiach

**Comment by benswift** [comment]
Thanks for adding me 😄  I'm not 100% sure when I'll get around to this issue yet, but now at least I can fix it in principle.

**Comment by 0x31** [comment]
I'm happy to update the command as well - but feel free to move the repo to the extemporelang org (I assume you have access to the repo settings now as well).

**Comment by benswift** [comment]
No, I don't have a settings tab (only the project owner does) so I can't transfer it. I think you can do it - you're a member of that org anyway.

**Comment by 0x31** [comment]
I don't have repo-creating permissions in the org. I've initiated a transfer of the repository to your own account, which should then let you transfer it to extemporelang.

**Comment by benswift** [comment]
oh my. what a complicated dance. Thanks Noah :)

**Comment by benswift** [comment]
aaaaand it's done.

So yeah, since the _acutal_ issue is the "change the defaults for the connect command thing" if you were still able to do that that'd be super. But again there's no pressure - if you can't do it then someone else will find the time at some stage.
